src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -353,20 +381,33 @@ namespace argos
    const CRange<Real> CPointMass3DDroneModel::THRUST_LIMIT = CRange<Real>(-15, 15);
    const CRange<Real> CPointMass3DDroneModel::ROLL_PITCH_LIMIT = CRange<Real>(-0.5, 0.5);
    const Real CPointMass3DDroneModel::XY_VEL_MAX = 1;
-   const Real CPointMass3DDroneModel::Z_VEL_MAX = 1;
+   const Real CPointMass3DDroneModel::Z_VEL_MAX = 0.05;
    /* PID coefficients */
    const Real CPointMass3DDroneModel::XY_POS_KP = 1;
    const Real CPointMass3DDroneModel::XY_VEL_KP = 3;
    const Real CPointMass3DDroneModel::YAW_KP = 13;
    const Real CPointMass3DDroneModel::YAW_KI = 0;
    const Real CPointMass3DDroneModel::YAW_KD = 8;
-   const Real CPointMass3DDroneModel::ALTITUDE_KP = 10;
+   const Real CPointMass3DDroneModel::ALTITUDE_KP = 5;
    const Real CPointMass3DDroneModel::ALTITUDE_KI = 0;
    const Real CPointMass3DDroneModel::ALTITUDE_KD = 6;   
    const Real CPointMass3DDroneModel::ROLL_PITCH_KP = 12;
    const Real CPointMass3DDroneModel::ROLL_PITCH_KI = 0;
    const Real CPointMass3DDroneModel::ROLL_PITCH_KD = 6;
    const Real CPointMass3DDroneModel::ROOT_TWO = std::sqrt(2.0);
+   /* sensor noise coefficents*/
+   const Real CPointMass3DDroneModel::MEAN_SENS = 0;
+   const Real CPointMass3DDroneModel::STD_GYRO_X = 0.67;
```

Comment:
You need to improve the naming of some of these variables, e.g., `STD_GYRO_X` should at least be `STD_DEV_GYRO_X